webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/server/server_details.html

```diff
@@ -18,7 +18,7 @@
                 <span class="ion ion-checkmark-round check-icon"></span>
               </label>
               <label for="typeQuery" class="form-control">
-                Include Rudder server components
+                Include Rudder root server
```

Comment:
It is not only root, but all node with a server component: so nodes having rudder-reports, rudder-db, rudder-webapp or rudder-server-root
But not the relays

Telling root is not correct - maybe something in the line
"Include Rudder server components (web interface, database)"

Reply:
nobody knows what "server components" are (I didn't think it's documented anywhere). I really think it should say "root". 

```
Include Rudder root server components (nodes with Rudder application or database)
```

Reply:
"application" is maybe too broad, though

Reply:
```
Include Rudder root server components (nodes with Rudder web application or database)
```

Reply:
![image](https://user-images.githubusercontent.com/44649/96509081-d2081100-125b-11eb-81dc-2dfe881e5c33.png)